streams-contrib/streams-provider-twitter/src/main/java/org/apache/streams/twitter/provider/TwitterFollowingProviderTask.java

```diff
@@ -141,7 +141,7 @@ else if( endpoint.equals("friends") )
             catch(Exception e) {
                 keepTrying += TwitterErrorHandler.handleTwitterError(client, e);
             }
-        } while (curser != 0 && keepTrying < 10);
+        } while (curser != 0 && keepTrying < 50);
```

Comment:
I think this should be left out of this PR in favor of making the TwitterErrorHandler respect the Twitter4j RateLimitStatus reset time. I'm volunteering to do that, so unless STREAMS-311 can't be fixed without it I'd say to put this back to 10.